app/components/Views/OnboardingSuccess/DefaultSettings/index.test.tsx

```diff
@@ -33,11 +32,11 @@ const mockNetworkName = 'Ethereum Main Network';
 
 describe('DefaultSettings', () => {
   it('should render correctly', () => {
```

Comment:
S: as you are here, maybe also rename the test to match guidelines:
```suggestion
  it('render matches snapshot', () => {
```